### PR TITLE
[ENDPOINT] OT226-62 Add Slides to public endpoint

### PR DESCRIPTION
**Pull request by Franckfer**
CARD [OT226-62](https://alkemy-labs.atlassian.net/browse/OT226-62)

<hr>

## Description

* AS a user
* I WANT to see Slides when I log in to the page
* TO be more informed about them.

### Acceptance criteria:

* In the Organization's public data endpoint, the list of Slides sorted by the "order" field should be added. 

* The Slides will be those belonging to the Organization, based on the organization_id field that must correspond to the Organization that is being returned.

<hr>

## Evidence

**Slides**
slidesIMG

### CODE REVIEW

**Review by nicoescudero — APPROVED** [review]
buen pr

**Review by afrivera — APPROVED** [review]
Buen PR

**Review by dangello-g — APPROVED** [review]
Bien

File: controllers/slides.js [hunk 1]
```diff
@@ -82,29 +82,4 @@ module.exports = {
...
-        body: response,
-      })
-    } catch (error) {
-      const httpError = createHttpError(
-        error.statusCode,
-        `[Error updating slide] - [Slide - PUT]: ${error.message}`,
-      )
-      next(httpError)
-    }
-  }),
```
Comment (gabodmc) [inline controllers/slides.js]: Por que se elimino todo esto?

**Review by gabodmc — CHANGES_REQUESTED** [review]
Solo un comentario

**Review by gabodmc — APPROVED** [review]

**Pull request merged**